administration/configuring-fluent-bit/classic-mode/configuration-file.md

```diff
@@ -1,96 +1,107 @@
+Bit allows the use one configuration file which works at a global scope and uses the
+defined [Format and Schema](format-schema.md).
+
+The main configuration file supports four sections:
+
+- Service
+- Input
+- Filter
+- Output
+
+It's also possible to split the main configuration file into multiple files using
+the Include File feature to include external files.
+
+## Service
+
+The `Service` section defines global properties of the service. The following keys
+are:
+
+| Key             | Description   | Default Value |
+| --------------- | ------------- | ------------- |
+| `flush`           | Set the flush time in `seconds.nanoseconds`. The engine loop uses a Flush timeout to define when it's required to flush the records ingested by input plugins through the defined output plugins.  | `1` |
+| `grace`           | Set the grace time in `seconds` as an integer value. The engine loop uses a Grace timeout to define wait time on exit. | `5` |
+| daemon          | Boolean. Set if Fluent Bit should run as a Daemon (background) or not. Allowed values are: `yes`, `no`, `on`, and `off`. When using a Systemd based unit as the one provided in FLuent Bit packages, don't turn on this option.  | `Off` |
+| `dns.mode`        | Set the primary transport layer protocol used by the asynchronous DNS resolver. Can be overridden on a per plugin basis. | `UDP` |
+| `log_file`        | Absolute path for an optional log file. By default all logs are redirected to the standard error interface (stderr). | _none_ |
+| `log_level`       | Set the logging verbosity level. Allowed values are: `off`, `error`, `warn`, `info`, `debug`, and `trace`. Values are cumulative. If 'debug' is set, it will include `error`, `warning`, `info`, and `debug`. Trace mode is only available if Fluent Bit was built with the _`WITH_TRACE`_ option enabled. | `info` |
+| `parsers_file`    | Path for a `parsers` configuration file. Multiple `Parsers_File` entries can be defined within the section. | _none_ |
+| `plugins_file`    | Path for a `plugins` configuration file. A `plugins` configuration file defines paths for external plugins. [See an example](https://github.com/fluent/fluent-bit/blob/master/conf/plugins.conf). | _none_ |
+| `streams_file`    | Path for the Stream Processor configuration file. [Learn more about Stream Processing configuration](../../../stream-processing/introduction.md). | _none_|
+| `http_server`     | Enable the built-in HTTP Server | `Off` |
+| `http_listen`     | Set listening interface for HTTP Server when it's enabled | `0.0.0.0` |
+| `http_port`       | Set TCP Port for the HTTP Server. | `2020` |
+| `coro_stack_size` | Set the coroutines stack size in bytes. The value must be greater than the page size of the running system. Setting the value too small (`4096`) can cause coroutine threads to overrun the stack buffer. The default value of this parameter shouldn't be changed. | `24576` |
```

Comment:
```suggestion
| `coro_stack_size` | Set the co-routines stack size in bytes. The value must be greater than the page size of the running system. Setting the value too small (`4096`) can cause co-routine threads to overrun the stack buffer. The default value of this parameter shouldn't be changed. | `24576` |
```

Reply:
I looked this one up and the common usage appears to be 'coroutines', no hyphen.